kitsune/sumo/urls.py

```diff
@@ -1,21 +1,14 @@
-from django.conf.urls import url, include
+from django.conf.urls import url
 from django.views.generic.base import RedirectView
 
 from watchman import views as watchman_views
 
 from kitsune.sumo import views
 
 
-services_patterns = [
-    url("^/monitor$", views.monitor, name="sumo.monitor"),
-    url("^/version$", views.version_check, name="sumo.version"),
-    url("^/error$", views.error, name="sumo.error"),
```

Comment:
Should we remove `error` from `kitsune/sumo/views.py` then?

Also will need to remove `ForceErrorTests` from `kitsune/sumo/tests/test_views.py`